common/networkpolicies/base/trainer-webhook-kubeflow-system.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: trainer-webhook
+  namespace: kubeflow-system
+spec:
+  podSelector:
+    matchExpressions:
+    - key: app.kubernetes.io/name
+      operator: In
+      values:
+      - trainer
+    - key: app.kubernetes.io/component
+      operator: In
+      values:
+      - manager
```

Comment:
can you find a more unique match than manager? this could clash with other deployments